Dockerfile

```diff
@@ -1,48 +1,70 @@
-FROM ubuntu:22.10
+FROM ubuntu:22.04
```

Comment:
22.10 isn't supported for ODBC 18 install: https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver16&tabs=ubuntu18-install%2Calpine17-install%2Cdebian8-install%2Credhat7-13-install%2Crhel7-offline